2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 182 Line 18_
```markdown
#### Example use case: Phone book / cell phone contacts: 

You want to repeatedly “look up” values that are associated with identifiers. E.g. your contacts list, which might pair a name (the `key`) with a phone-number (the `value`). Want to look up a friends phone number? Just type their name (the `key`) and you get their phone number (the `value`) instantly.

##### Dictionary Syntax: 
* Dictionaries can be created using curly braces `{}`, or with the `dict()` constructor:
* Create an empty dictionary: `empty_dict = {}`, or equivalently, `empty_dict = dict{}`
* Create a dictionary with data: `emails={'aakash':'aaahamed@calpoly.edu', 'andrew':'ajnisbet@gmail.com'}`
```
Can you change my email to `andrew@ajnisbet.com` (it's more pro! and it's my public-facing one) 🙏

Reply:
done!